jobs/JGDAS_GLOBAL_OCEAN_ANALYSIS_RUN

```diff
@@ -0,0 +1,90 @@
+#!/bin/bash
+export STRICT="NO"
+source "${HOMEgfs}/ush/preamble.sh"
+
+#############################
+# Source relevant config files
+#############################
+export EXPDIR=${EXPDIR:-${HOMEgfs}/parm/config}
+configs="base ocnanal ocnanalrun"
+config_path=${EXPDIR:-${PACKAGEROOT}/gfs.${gfs_ver}/parm/config}
+for config in ${configs}; do
+    . "${config_path}"/config."${config}"
+    status=$?
+    [[ ${status} -ne 0 ]] && exit "${status}"
+done
+
+
+##########################################
+# Source machine runtime environment
+##########################################
+. "${HOMEgfs}"/env/"${machine}".env ocnanalrun
+status=$?
+[[ ${status} -ne 0 ]] && exit "${status}"
+
+
+##############################################
+# Obtain unique process id (pid) and make temp directory
+##############################################
+export pid=${pid:-$$}
+export outid=${outid:-"LL${job}"}
+
+export DATA=${DATA:-${DATAROOT}/ocnanal_${CDATE}}
+mkdir -p "${DATA}"
+cd "${DATA}" || (echo "${DATA} does not exist. ABORT!"; exit 1)
+
+
+##############################################
+# Run setpdy and initialize PDY variables
+##############################################
+export cycle="t${cyc}z"
+setpdy.sh
+. ./PDY
+
+
+##############################################
+# Determine Job Output Name on System
+##############################################
+export pgmout="OUTPUT.${pid}"
+export pgmerr=errfile
+
+
+##############################################
+# Set variables used in the script
+##############################################
+
+export CDUMP=${CDUMP:-${RUN:-"gfs"}}
+export COMPONENT=${COMPONENT:-ocean}
+
+##############################################
+# Begin JOB SPECIFIC work
+##############################################
+
+export COMOUT=${COMOUT:-${ROTDIR}/${CDUMP}.${PDY}/${cyc}/${COMPONENT}}
+
+###############################################################
+# Run relevant script
+
+EXSCRIPT=${GDASPREPPY:-${HOMEgfs}/sorc/gdas.cd/scripts/exgdas_global_marine_analysis_run.sh}
+${EXSCRIPT}
+status=$?
+[[ ${status} -ne 0 ]] && exit "${status}"
+
+##############################################
+# End JOB SPECIFIC work
+##############################################
+
+##############################################
+# Final processing
+##############################################
+if [[ -e "${pgmout}" ]] ; then
+  cat "${pgmout}"
+fi
+
+##########################################
+# Remove the Temporary working directory
+##########################################
+cd "${DATAROOT}" || exit 1
+[[ ${KEEPDATA} = "NO" ]] && rm -rf "${DATA}"
```

Comment:
Don't want this one either I don't think unless you have nothing to archive/save from this job back to the ROTDIR.

Reply:
@CoryMartin-NOAA , unless I messed up (very likely), the analysis and increment are dumped in `COMOUT` (==`ROTDIR`?)  ... And as I write this, I realize I forgot to do the same for `diag`. Is it OK to have symlinks to directories in `COMOUT`/`ROTDIR` in which we directly write what we want to keep?

Reply:
for now it is, but 'eventually' it is not. NCO wants no symlinks, and it makes sense from a solid state drive for run / spinning disk for storage perspective. What my plan is/was for the aerosol tasks (see PR#1106) is to copy everything I want to save from  $DATA to $COMOUT as part of the finalize job after the run job.

Reply:
Sounds good @CoryMartin-NOAA , I'll do this in a subsequent PR if that's OK with you (I'm just being polite and respectful in exchange for a green check). My next step is to reconcile my cycling branch with the changes here and make it work again, which I'm pretty sure will require some adjustment/bug fixes in the GDASApp and the g-w. Looking forward to it!